.github/workflows/build.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           make test
 
       - name: Upload coverage
-        uses: codecov/codecov-action@v5
+        uses: codecov/codecov-action@v4
```

Comment:
Why this version is downgraded ?

Reply:
I faced this issue: https://github.com/codecov/codecov-action/issues/1699  which is still open on their side. I thought we can upgrade when it is fixed.